### PR TITLE
Eos

### PR DESCRIPTION
**Pull request by markrypto**

### CODE REVIEW

File: packages/hdwallet-keepkey/src/eos.ts [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+  if (c.charCodeAt(0) >= "a".charCodeAt(0) && c.charCodeAt(0) <= "z".charCodeAt(0)) {
+    return ((c.charCodeAt(0) - "a".charCodeAt(0)) + 6)
+  }
+  if (c.charCodeAt(0) >= "1".charCodeAt(0) && c.charCodeAt(0) <= "5".charCodeAt(0)) {
+    return ((c.charCodeAt(0) - "1".charCodeAt(0)) + 1)
+  }
+  return 0
+}
+
+function nameToNumber(name:string) : string {
```
Comment (elmutt) [inline packages/hdwallet-keepkey/src/eos.ts:44]: Whats this do?

File: packages/hdwallet-keepkey/src/eos.ts [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+    } else { // last iteration
+        c = c.and(Long.fromString("0f", true, 16))
+    }    
+      value = value.or(c)
+  }
+  return value.toString()
+}
+
+function assetToNumber(asset:string): [number, number] {
```
Comment (elmutt) [inline packages/hdwallet-keepkey/src/eos.ts:66]: whats this do?

File: packages/hdwallet-keepkey/src/eos.ts (hunk 1 above)
Comment (markrypto) [reply packages/hdwallet-keepkey/src/eos.ts:44]: Converts the alphanumeric name of an action to a format that is appropriate for signing. This was derived from the python-keepkey tests for eos that came from trezor.

File: packages/hdwallet-keepkey/src/eos.ts (hunk 2 above)
Comment (markrypto) [reply packages/hdwallet-keepkey/src/eos.ts:66]: Same as the nameToNumber(), converts the asset name to a number for signing. Derived from the trezor reference for testing.

**Pull request closed without merging**